<launch>
    <param name="xlims" value="[0,20]"/>
    <param name="ylims" value="[0,20]"/>
    <param name="resolution" value="0.4"/>
    <param name="height" value="3"/>
    <param name="sigma_x" value="2"/>
    <param name="sigma_y" value="5"/>
    <param name="sensor_accuracy" value="0.9"/>

    <node name="mapping" pkg="plume_gsl" type="mapping.py" output="screen">
        # Sensor params
        <param name="verbose" value="false"/>
        <param name="fixed_frame" value="map"/>
        <param name="anemometer_frame" value="anemometer_frame"/>
        <param name="sensor_frame" value="mox_frame"/>
        <param name="use_service_for_gas" value="false"/>
    </node>

    <node pkg="plume_gsl" type="source_from_map.py" name="source_from_map"/>
    <!-- <node pkg="rqt_plot" type="rqt_plot" name="Concentration" args="/concentration/data"/> -->
    <!-- <node pkg="rosbag" type="record" name="rosbagger" args="record -o $(find plume_tools)/bagfiles/ /base_pose_ground_truth /concentration /Anemometer/WindSensor_reading /max_probability"/> -->
    
    <!-- The robot starts moving to this point. -->
    <!-- Comment this line to prevent the robot motion -->
    <!-- <node pkg="crazyflie_control" type="mover.py" name="mover" args="15 10"/>  -->
</launch>
